### DISCUSSION TITLE
Support global 7TV emotes

### DISCUSSION THREAD

**Discussion by NBruderman** [night/betterttv#5886] [Ideas]
The addition of 7TV to the extension is welcomed, but for some reason it only includes the channel emotes,
and not the global emotes, which are also being used quite often,
Any chance of implementing the global emotes too?

**Comment by jjjj222** [comment]
I migrated from 7tv back to Bttv when I heard it supports 7tv emotes now.... yet found out some emotes are missing so have to switch back to 7tv. Quite disappointing.

**Comment by NBruderman** [comment]
Just to be clear, all the offensive global emotes are long gone, and some streamers (xqc, for example) removed the clap emotes, as it's a global 7TV emote, and relying on that

**Comment by night** [comment]
@EdVraz to reply to your comment from the similar issue, we haven't gotten around to this yet. a PR may be welcome assuming they have cleaned up their global emotes and intend to keep them clean

**Reply by NBruderman** [reply]
Unfortunately, I don't have the coding skills to make such PR... however, I can confirm that 7TV fully cleaned their global emotes from any offensive emotes, and they're intending to keep it clean, as it's now widely adopted around twitch.

**Reply by sillyElvy** [reply]
I think I got it, just have to check a few things

**Reply by sillyElvy** [reply]
#5903

**Comment by night** [comment]
This will be out in the next release.
